shiftfs/shiftfs.go

```diff
@@ -167,21 +176,21 @@ func runShiftfsCheckOnHost(dir string, checkOnOverlayfs bool) (bool, error) {
 			os.Exit(1)
 		}
 		if err := Mount(testDir, testDir); err != nil {
-			os.Exit(1)
+			os.Exit(2)
```

Comment:
nit: I would have added descriptive logs instead of relying on differentiated exit() values.

Reply:
It was harder to get logs because this code snippet is running inside a child process of sysbox-mgr which is in a user-namespace. I didn't want to go through the trouble of implementing the mechanism to pass the logs from the child process into the sysbox-mgr, so the latter could output them as usual.